titus-server-federation/src/test/java/com/netflix/titus/federation/service/GrpcFallbackTest.java

```diff
@@ -0,0 +1,115 @@
+package com.netflix.titus.federation.service;
+
+import com.netflix.titus.grpc.protogen.JobDescriptor;
+import com.netflix.titus.grpc.protogen.JobId;
+import com.netflix.titus.grpc.protogen.JobManagementServiceGrpc;
+import com.netflix.titus.runtime.endpoint.common.grpc.GrpcUtil;
+import io.grpc.stub.StreamObserver;
+import io.grpc.testing.GrpcServerRule;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.jupiter.api.BeforeEach;
+import rx.Observable;
+
+import java.util.Optional;
+
+import static com.netflix.titus.runtime.endpoint.common.grpc.GrpcUtil.createRequestObservable;
+import static org.assertj.core.api.Assertions.assertThat;
+
+class JobManagementServiceWithUnimplementedInterface extends JobManagementServiceGrpc.JobManagementServiceImplBase {
+    // All interface methods return UNIMPLEMENTED status.
+}
+
+class JobManagementServiceWithSlowMethods extends JobManagementServiceGrpc.JobManagementServiceImplBase {
+    public long createCount = 0;
```

Comment:
minor: set the field as `volatile` to avoid it biting us in the future in case we stop using `directExecutor()`